### PR TITLE
Add tools description

### PR DESCRIPTION
**Pull request by thibaudrobin**
# Description

Hello there ! What about some tools description ?

# Related issues

Roadmap : https://github.com/orgs/ThePorgs/projects/1/views/1?pane=issue&itemId=21406532

# Point of attention

I hope there is no mistake !

### CODE REVIEW

**Comment by ShutdownRepo** [comment]
Cancelled checks because the target branch was wrong. Fixed branch, can you fix the conflicts?

File: sources/install.sh [hunk 1]
```diff
@@ -1238,15 +1349,15 @@ function install_bloodhound_old_v3() {
   unzip /tmp/BloodHound-linux-x64.zip -d /opt/tools/
   mv /opt/tools/BloodHound-linux-x64 /opt/tools/BloodHound3
   rm /tmp/BloodHound-linux-x64.zip
-}
+}#
```
Comment (ShutdownRepo) [inline sources/install.sh:1352]: why adding a `#` here?

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]
There are many links that are wrong. Below are some examples. Can you review your PR and correct them?

File: sources/install.sh (hunk 1 above)
Comment (thibaudrobin) [reply sources/install.sh:1352]: }# : this indicates this tool is duplicated and his version should not be documented because it would cause a duplication

Comment (ShutdownRepo) [reply sources/install.sh:1352]: alright, we should probably add a comment indicating that then

**Comment by Dramelac** [comment]
Thank you for the help but there are still a lot of broken/wrong links ... 

I'm putting this PR on hold. Let us know when it is fully ready for review and validation.

**Comment by thibaudrobin** [comment]
I'm working on it

**Comment by thibaudrobin** [comment]
Good ?

**Comment by thibaudrobin** [comment]
Conflict solved

**Comment by ShutdownRepo** [comment]
Before we go into review mode, did you check all links and descriptions to make sure they match?

**Comment by thibaudrobin** [comment]
yep

**Pull request merged**